src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -91,6 +91,9 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
       }),
     [infraEnv, cluster, pullSecret, managedDomains, ocpVersions, search],
   );
+  initialValues.openshiftVersion =
```

Comment:
What's the difference between this and the code in `getDefaultOpenShiftVersion`?

It's preferable that the function ``ClusterDetailsService.getClusterDetailsInitialValues()` already sets the correct values to all fields, including `penShiftVersion`. And also that it's memoized by being inside the `React.memo` block.

Reply:
getDefaultOpenShiftVersion only returns 4.11, 4.8, 4.9 and not 4.11.5, 4.8.1 or 4.9.2
I don't touch this function because is used in CIM side
Maybe the best is to implement an OCM function (getDefaultOCMOpenShiftVersion)
@celdrake WDYT?

Reply:
We may need to check with them, but the "parent" version doesn't seem right. As it's setting on the form a value which is not correct (users select a specific version, not the parent version, and there can be more than 1 versions of the same parent to choose from)

Reply:
I have tried it with "master" and the versions that are sent are the parent version (not the specific):
![image](https://user-images.githubusercontent.com/11390125/192944538-5ff4350f-108a-4c56-a642-d01bb72855bd.png)
![image](https://user-images.githubusercontent.com/11390125/192944560-e2660823-cdcb-401f-9256-3aa1d9f7c69c.png)

Reply:
we've been looking at this, and it looks like despite it's not consistent (it doesn't always send the short or the long version), it's correct.